sorts/test/bogo_sort.test.ts

```diff
@@ -0,0 +1,16 @@
+import { bogoSort } from '../bogo_sort';
+
+describe('BubbleSort', () => {
+	it('should return the correct values for test case 1', () => {
```

Comment:
These test case names aren't useful. See [the contribution guidelines](https://github.com/TheAlgorithms/TypeScript/blob/master/CONTRIBUTING.md#writing-good-tests). Please simply remove the `it` blocks or use `each`.

Reply:
Thanks for pointing it out, I changed the tests to use test.each